src/Pipfile

```diff
@@ -23,6 +23,7 @@ requests = "*"
 django-fsm = "*"
 django-phonenumber-field = {extras = ["phonenumberslite"], version = "*"}
 boto3 = "*"
+typing-extensions ='*'
```

Comment:
This must be added to packages (not dev-packages) to avoid a [CI failure ](https://github.com/cisagov/getgov/actions/runs/5147786877/jobs/9268629945)

Reply:
Not sure exactly why it's now needed when it wasn't before. There's a [small change to this dep in the requirements.txt](https://github.com/cisagov/getgov/pull/668/files#diff-0a0827574cc2d3aac80f74096dd6d1e871b83efae03aa087c2c8e8f1e9187d26R48)